### ISSUE TITLE
feedback: Option to go back to the dedicated chat window instead of sidebar

### ISSUE THREAD

**Issue by endolith** [sourcegraph/cody#5032]
### Version

 v1.28.0

### Areas for Improvement

- [X] UI/UX
- [ ] Onboarding
- [ ] Docs
- [ ] Chat
- [ ] Commands
- [ ] Context
- [ ] Response Quality
- [ ] Other

### What needs to be improved? Please describe how this affects the user experience and include a screenshot.

"Cody for VS Code v1.28: Introducing the unified Cody sidebar" changed the chat functionality so it's no longer in a dedicated pane on the right side of the IDE, but now is in the sidebar on the left, which is unusably narrow and annoying to use:

![image](https://github.com/user-attachments/assets/9bebc386-2980-42a2-9f9b-2db9d85ada1c)

Codieum uses the sidebar too, which is why I ditched it and went back to Cody, but now Cody has done the same thing.

The sidebar can be resized, but then it's resized for every other use of the sidebar, too, which is not desirable.  File explorer should be narrow, extensions list should be narrow, etc.

### Describe the solution you'd like to see

There should be an option to always open new chats in a dedicated pane, like it used to do.

**Comment by jdorfman** [comment]
Hey @endolith you can do that by click the following:
<img width="395" alt="Zight 2024-07-29 at 8 00 18 AM png" src="https://github.com/user-attachments/assets/01c40dd7-120d-4e87-9f7a-1e55efa6024b">

LMK if that works or not

**Comment by endolith** [comment]
@jdorfman Yes I've been clicking that, but I still need to do it each time.  Conversations always start in the sidebar, which is not desirable.

**Comment by jdorfman** [comment]
@endolith noted. I will make sure the product team sees this in the weekly community feedback report (next Monday).

**Comment by jiri-bajer** [comment]
As a temporary workaround it is at least possible to configure a keyboard shortcut for moving the chat into editor panel: `Cody: Move Chat to Editor Panel` (`cody,chat.moveToEditor`). The left sidebar really is too thin and isn't a good place for the chat. The UI of the sidebar is IMO considerably worse than before and screen estate is wasted by help and actions which can be short-cutted.

**Comment by endolith** [comment]
(I can't even see the "Open in Editor" button when the file pane is narrow:)
![image](https://github.com/user-attachments/assets/6ab79730-73a4-43b9-aaf1-7c7b5ed26f73)
![image](https://github.com/user-attachments/assets/571cac2c-b027-4a23-9bd0-a992d5d849b0)

**Comment by jiri-bajer** [comment]
And there is also a `Cody > Chat: Default Location` (`cody.chat.defaultLocation`) which can be set to `editor` or `sticky` to address the issue.

**Comment by endolith** [comment]
I have this enabled:

![image](https://github.com/user-attachments/assets/0082d109-6095-441a-b54c-e59b7d5de1a7)

but it still creates a chat in the sidebar if I right-click and select Cody → Add File to Cody Chat.
